resume.md

```diff
@@ -1,49 +1,64 @@
-# FIRST LAST NAME
+# Tejesh Bhaumik
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 7890 | tejeshbhaumik@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+| github.com/Tejesh2001 | linkedin.com/TejeshBhaumik
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**UIUC** | B.S. of Computer Science + Advertising... *May 2023*
+* **Coursework**: Data Structures, Algorithms, Advanced Linear Algebra, Statistics, Data Science, Probability, Brand Strategy.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C++, SQL, NodeJs, Android
+* **Libraries/Frameworks**: Pandas, Numpy, Scipy, Sympy
+* **Version Control**: Git, GithubDesktop
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Develop the UI of a tracer app using Recycler and Card views in Android studio
+* Implement the authentication protocol using Google OAuth2
+* The app is used for tracking data for people within a closed location such as a department store and targeting preferences based on their location
+
+**Backend Developer**: Ensemble Education | Remote *August 2020 - October 2020*
+* Implemented Google OAuth2 and Facebook's login using the Graph API.
+* Implemented a file system through a closure table in PostGresSql
+* Assisted in the development of the AI module using scikit-learn to calculate expectation values and build a linear regression model
+
+**Software Research Assistant**: UIUC advertising department   | Remote *July 2020 - August 2020*
+* Developed an AI chatbot with DialogFlow in Node.js with a user interface in HTML/CSS. Has a user base of 200 surveyors.
```

Comment:
still do not understand what this project does and why someone would need a chatbot to solve the problem.  Unclear.  First bullet point should explicitly say what the problem / purpose of this project is + how you took a role as a contrbutor to this project